lm75bd/lm75bd.c

```diff
@@ -26,8 +26,24 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  // declare error code variable
+  error_code_t errCode;
   
+  // set pointer register to 'temperature sensor'
+  uint8_t writeBuf[] = {0};
```

Comment:
since you know this should only be 1 uint8_t long, its best to explicitly initialize it to that size for readability